test/docbook-xref.native

```diff
@@ -4,7 +4,7 @@ Pandoc (Meta {unMeta = fromList [("title",MetaInlines [Str "An",Space,Str "Examp
 ,BulletList
  [[Para [Str "A",Space,Str "straight",Space,Str "link",Space,Str "generates",Space,Str "the",SoftBreak,Str "cross-reference",Space,Str "text:",Space,Link ("",[],[]) [Str "The",Space,Str "Second",Space,Str "Chapter"] ("#ch02",""),Str "."]]
  ,[Para [Str "A",Space,Str "link",Space,Str "to",Space,Str "an",Space,Str "element",Space,Str "with",Space,Str "an",SoftBreak,Str "XRefLabel:",SoftBreak,Link ("",[],[]) [Str "Chapter",Space,Str "the",Space,Str "Third"] ("#ch03",""),Str "."]]
- ,[Para [Str "A",Space,Str "link",Space,Str "with",Space,Str "an",SoftBreak,Str "EndTerm:",SoftBreak,Link ("",[],[]) [Str "Chapter",Space,Str "4"] ("#ch04",""),Str "."]]
+ ,[Para [Str "A",Space,Str "link",Space,Str "with",Space,Str "an",SoftBreak,Str "EndTerm:",SoftBreak,Link ("",[],[]) [Str "ch04short"] ("#ch04",""),Str "."]]
```

Comment:
This doesn't look right, does it?

Reply:
Also, maybe add a test that tests explicitly for the case being fixed?

Reply:
Yes, after carefully reading the [specification](https://tdg.docbook.org/tdg/5.1/xref.html) I find that #6619 isn't a bug. And pandoc is doing as the specification says.

In particular:

```
<simpara>See link with endterm: <xref linkend="SEC-Definitions" endterm="link"/>.</simpara>
```

`link` should be the id of the element whose content should be used for the link text.

> If the endterm attribute is specified on xref, the content of the element pointed to by endterm will be used as the text of the cross-reference.

So this pull request can be discarded.